packages/opentelemetry/src/metrics.test.ts

```diff
@@ -46,13 +46,14 @@ describe('instrumentedWithMetrics', () => {
     await waitForMetricsExport(exporter);
     const exportedMetrics = exporter.getMetrics();
     const { scope, metrics } = exportedMetrics[0].scopeMetrics[0];
-    const [counterMetric, histogramMetric] = metrics;
 
-    assert.equal(scope.name, 'test');
+    // We won't see an exported metric for the error counter because the
+    // Metrics SDK no longer exports metrics with no data points.
+    // https://github.com/open-telemetry/opentelemetry-js/pull/4135
+    assert.lengthOf(metrics, 1);
+    const [histogramMetric] = metrics;
 
-    assert.ok(counterMetric);
-    assert.equal(counterMetric.descriptor.name, 'test.error');
-    assert.lengthOf(counterMetric.dataPoints, 0);
```

Comment:
The counter metric is no longer reported: https://github.com/open-telemetry/opentelemetry-js/pull/4135